lib/std/net.zig

```diff
@@ -25,20 +38,20 @@ pub const Address = extern union {
     /// IPv6 link-local unix addresses.
     pub fn parseIp(name: []const u8, port: u16) !Address {
         if (parseIp4(name, port)) |ip4| return ip4 else |err| switch (err) {
-            error.Overflow,
-            error.InvalidEnd,
-            error.InvalidCharacter,
-            error.Incomplete,
-            error.NonCanonical,
+            IPv4ParseError.Overflow,
+            IPv4ParseError.InvalidEnd,
+            IPv4ParseError.InvalidCharacter,
+            IPv4ParseError.Incomplete,
+            IPv4ParseError.NonCanonical,
```

Comment:
Please don't change these.

Reply:
Sure, will roll those back. 👍 
I was questioning whether to change it or not, AFAIK it doesn't affect runtime, so, a style choice. I was looking at other std libs to see if there is a clear convention regarding this, but seems both `error.SomeError` and `SomeErrorSet.SomeError` are used throughout the codebase.
Or is there some edge-cases where two are distinct? Some compiler logs I guess?

Reply:
`SomeErrorSet.SomeError` has the type `SomeErrorSet`, `error.SomeError` has the type `error{SomeError}` which implicitly casts to `SomeErrorSet`. `error.SomeError` is preferred because it makes it obvious that the value is an error.